oscps-lib/src/blocks.rs

```diff
@@ -12,7 +12,8 @@ use uom::si::f64::{Energy};
 use uom::si::f64::{Mass};
 use uom::si::mass::kilogram;
 use uom::si::energy::joule;
-use crate::connector;
+use crate::component::ChemicalIdentifier;
+use crate::{component, connector};
 use once_cell::sync::Lazy;
 
 //Initiallizing a global variable for the tolerance for the energy balance
```

Comment:
Some comments do not follow guidelines in https://doc.rust-lang.org/rust-by-example/meta/doc.html. Future pull request should correct this documentation style